fastlane/spec/actions_specs/get_version_number_action_spec.rb

```diff
@@ -3,26 +3,41 @@
     describe "Get Version Number Integration" do
       require 'shellwords'
 
-      path = File.absolute_path("./fastlane/spec/fixtures/actions/get_version_number/get_version_number/")
+      xcodeproj_dir = File.absolute_path("./fastlane/spec/fixtures/actions/get_version_number/")
+      xcodeproj_filename = "get_version_number.xcodeproj"
 
       it "gets the correct version number for 'TargetA'", requires_xcodeproj: true do
         result = Fastlane::FastFile.new.parse("lane :test do
-          get_version_number(xcodeproj: '#{path}', target: 'TargetA')
+          get_version_number(xcodeproj: '#{xcodeproj_dir}', target: 'TargetA')
+        end").runner.execute(:test)
+        expect(result).to eq("4.3.2")
+      end
+
+      it "gets the correct version number for 'TargetA' using xcodeproj_filename without extension", requires_xcodeproj: true do
+        result = Fastlane::FastFile.new.parse("lane :test do
+          get_version_number(xcodeproj: '#{xcodeproj_dir}', target: 'TargetA')
+        end").runner.execute(:test)
+        expect(result).to eq("4.3.2")
+      end
```

Comment:
Are these two tests the same? 🤔

Reply:
Oops! Leftovers from the old tests I had written 😇 nice catch!

Reply:
Deleted the 2nd one since it's the one that had the wrong description 😁 Lemme know if there's anything else @joshdholtz ! 🙏